### ISSUE TITLE
Change in test files

### ISSUE THREAD

**Issue by rweigel** [hapi-server/server-nodejs#22]
The documentation describes several configuration options. In the changes from the schema-fixes, Example4.json no longer uses a command line command to generate metadata. The metadata is inline. Probably there are other similar changes.

All of the example files should do the same thing as they did before - only the node name should have changed from "command" to "command_...". The contents of the node should not change for the purpose of making the tests pass.

After fixing this, create a test that verifies that the metadata that existed before the schema-fixes merge will still pass the tests after the schema-fixes merge. The test should do something like (not tested, so probably won't work)

```
mv metdaata metadata-current
mkdir tmp
cd tmp
git pull ... repository before merge
mv server-nodejs/metadata ..
cd ..
npm test
rm -rf tmp && rm -rf metadata && mv metadata-current metadata
```

**Comment by rweigel** [comment]
Make sure that you include the above set of test commands in a shell script in the branch that you push so that we can execute it and have it run the tests on the old metadata.

**Comment by rweigel** [comment]
Closed by 1e032e1
